pinot-tools/src/main/java/org/apache/pinot/tools/tuner/strategy/OLSAccumulator.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.tools.tuner.strategy;
+
+import io.vavr.Tuple2;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+
+/**
+ * An implementation of {@link AbstractAccumulator}, to implement minimum pulling for {@link OLSAnalysisImpl}
+ */
```

Comment:
What does OLS mean?

Reply:
OLS is Ordinary Least Squares regression, I will change the naming